test-cargo-miri/cdylib/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use byteorder::{BigEndian, ByteOrder};
 
-#[no_mangle]
-extern "C" fn use_the_dependency() {
+#[unsafe(no_mangle)]
+unsafe extern "C" fn use_the_dependency() {
```

Comment:
Why did you make this function `unsafe`?

Reply:
Fair point, it's not required indeed.